TeacherWorkout.Api/GraphQL/TeacherWorkoutMutation.cs

```diff
@@ -50,6 +56,31 @@ public class TeacherWorkoutMutation : ObjectGraphType<object>
                     var input = context.GetArgument<ThemeUpdateInput>("input");
                     return updateTheme.Execute(input);
                 });
+
+
+            Field<SingleUploadPayloadType>("singleUpload")
+                .Argument<NonNullGraphType<UploadGraphType>>(Name = "file")
+                .Resolve(context =>
+                {
+                    var file = context.GetArgument<IFormFile>("file");
+
+                    if (file.Length > 5 * 1024 * 1024)
```

Comment:
It's better for such values to keep them in config files. I don't know how to access the config from the mutation file, but maybe it's a topic for a new issue to open.